build.gradle

```diff
@@ -86,12 +86,4 @@ subprojects {
     test {
         useJUnitPlatform()
     }
-
-    compileJava {
-        options.encoding = "UTF-8"
-    }
-
-    compileTestJava {
-        options.encoding = "UTF-8"
-    }
```

Comment:
It is not normal that you have to fix this, do a rebase.

Reply:
it is, I moved it to a different pr

Reply:
but yeah I need to rebase